src/SqlConnectionConfig.ts

```diff
@@ -56,6 +52,31 @@ export default class SqlConnectionConfig {
         return auth?.replace(/\s/g, '').toLowerCase();
     }
 
+    /**
+     * Returns the connection string escaped by double quotes.
+     */
+    public get EscapedConnectionString() : string {
```

Comment:
Can we do effectively the same thing with `_parsedConnectionString` and iterate through keys/values so that we don't need to parse a second time?

Reply:
I did think about that but the issue with the key/value pair is the keys may have changed (e.g. Server becomes 'data source') and we'd still need to escape each value again. Since the full connection string is only used by sqlpackage, I wanted it to be as close to the one user supplied without changing it too much.

Reply:
Ahh, I see. Ok, I can live with that for sure. Can you put a comment here describing this decision?

Reply:
Added a comment